src/main/java/com/conveyal/datatools/manager/auth/Auth0Connection.java

```diff
@@ -132,8 +134,11 @@ private static void remapTokenValues(Map<String, Object> jwt) {
      * tables in the database.
      */
     public static void checkEditPrivileges(Request request) {
-        if (authDisabled()) {
-            // If in a development environment, skip privileges check.
+        if (authDisabled() || inTestingEnvironment) {
+            // If in a development or testing environment, skip privileges check. This is done so that basically any API
+            // endpoint can function.
+            // TODO: make unit tests of the below items or do some more stuff as mentioned in PR review here:
+            // https://github.com/conveyal/datatools-server/pull/187#discussion_r262714708
```

Comment:
I think it's often good to link to issues in the code, but in this case I think we should just state what we want to do directly in the `TODO`: `TODO: make unit tests of the below permissions checks. This would require creating one or more complete mock users (Auth0UserProfile) that contains a full set of permissions.`